tox.ini

```diff
@@ -34,3 +35,11 @@ flake8-ignore =
 flake8-select = F
 flake8-max-complexity = 18
 flake8-max-line-length = 88
+
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38, mypy
```

Comment:
can you explain mypy wrt 3.8 only plz?

Reply:
This comes from the [`tox-gh-actions`](https://github.com/ymyzk/tox-gh-actions) page. I believe it's enough to put it along a single version.